packages/frameworks/src/frameworks.ts

```diff
@@ -1691,7 +1691,7 @@ export const frameworks = [
     defaultVersion: '0.13.0', // Must match the build image
   },
   {
-    name: 'Hydrogen',
+    name: 'Hydrogen (v1)',
```

Comment:
@TooTallNate This doesn't follow the naming convention of others.

For example, we have `Docusaurus 1` / `Docusaurus 2`